app/controllers/support_interface/multiple_provider_users_controller.rb

```diff
@@ -0,0 +1,119 @@
+module SupportInterface
+  class MultipleProviderUsersController < SupportInterfaceController
+    def new
+      @provider = Provider.find(provider_id_param)
+      @form = MultipleProviderUsersForm.new(
+        state_store: multiple_provider_user_store,
+        provider_id: provider_id_param,
+      )
+    end
+
+    def create
```

Comment:
'create' may be the wrong name for this action as it is only persisting to Redis not the db ... 
In fact, most of the the actions in this controller could possibly do with better names. Any suggestions welcome!

Reply:
Summary of actions

- 'new' is the ProviderUser upload form (CSV)
- 'create' stores the ProviderUsers to the state store
- 'new_user_permissions' is the permissions form for a single ProviderUser
- 'upload_provder_permissions' stores the ProviderUser's permissions in the state store
- 'check' is the review page (note the prototype specifically includes 'check' in the URL)
- 'add_users' persists the ProviderUsers and their permissions to the db

Reply:
Also welcome suggestions / comment re: naming of routes